tests/consensus_tests/test_failed_snapshot_recovery.py

```
@@ -95,6 +95,7 @@ def remove_file_from_tar(original_tar, file_to_remove, new_tar):
 
 # The test validates that a node can recover from a corrupted snapshot
 def test_failed_snapshot_recovery(tmp_path: pathlib.Path):
+    assert False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)